### PR TITLE
Update Commenting Systems, CI/CD, Dev blogs

### PR DESCRIPTION
**Pull request by dmitry-konchalenkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1452,6 +1453,7 @@ Table of Contents
   * [BearBlog](https://bearblog.dev/) - Minimalist, and Markdown powered blog and website builder.
   * [Dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
   * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
+  * [JetBrains Blog](https://blog.jetbrains.com/) — Industry and developer tools news by JetBrains.
```
Comment (ripienaar) [inline README.md:1456]: this does not really belong on this list

File: README.md [hunk 2]
```diff
@@ -1461,6 +1463,7 @@ Table of Contents
   * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience.
   * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
   * [Disqus](https://disqus.com/) - Disqus is a networked community platform used by hundreds of thousands of sites all over the web.
+  * [Remark42](https://remark42.com/) - Privacy-focused lightweight commenting engine.
```
Comment (ripienaar) [inline README.md:1466]: This is not a SaaS its self hosted, so not suited to the list

**Review by ripienaar — CHANGES_REQUESTED** [review]

**Comment by dmitry-konchalenkov** [comment]
Hi @ripienaar,
Thanks, fixed the additions as per you request! 

Cheers,
Dmitry

**Comment by ripienaar** [comment]
thanks a lot

**Pull request merged**